ern-composite-gen/src/generateComposite.ts

```diff
@@ -573,6 +580,34 @@ async function createCompositeBabelRc({ outDir }: { outDir: string }) {
   )
 }
 
+async function createMetroConfigJs({ outDir }: { outDir: string }) {
+  return fileUtils.writeFile(
+    path.join(outDir, 'metro.config.js'),
+    `const blacklist = require('metro-config/src/defaults/blacklist');
+module.exports = {
+  resolver: {
+    blacklistRE: blacklist([
+      // Ignore IntelliJ directories
+      /.*\\.idea\\/.*/,
+      // ignore git directories
+      /.*\\.git\\/.*/,
+      // Ignore android directories
+      /.*\\/app\\/build\\/.*/      
+    ])
+  },
+  transformer: {
+    getTransformOptions: async () => ({
+      transform: {
+        experimentalImportSupport: false,
+        inlineRequires: false,
+      },
+    }),
+    assetPlugins: ['ern-bundle-store-metro-asset-plugin'],
+  },
+};`
```

Comment:
Can you also insert one newline character after the semicolon here? Otherwise, the generated file will have no newline at end of file.